PyInstaller/hooks/hook-itk.py

```diff
@@ -0,0 +1,7 @@
+from PyInstaller.utils.hooks import collect_data_files
```

Comment:
Please add the standard copyright header

Reply:
Addressed in updated PR.